instrumentation/opentelemetry-instrumentation-pymemcache/src/opentelemetry/instrumentation/pymemcache/__init__.py

```diff
@@ -115,7 +114,7 @@ def wrapper(wrapped, instance, args, kwargs):
 @_with_tracer_wrapper
 def _wrap_cmd(tracer, cmd, wrapped, instance, args, kwargs):
     with tracer.start_as_current_span(
-        _CMD, kind=SpanKind.INTERNAL, attributes={}
+        cmd, kind=SpanKind.CLIENT, attributes={}
     ) as span:
```

Comment:
From before, does pymemcache not have the concept of `name`?

Reply:
I am not sure if I understood you correctly, but memcache supports this list of [commands](https://github.com/memcached/memcached/wiki/Commands). We are using the command as a span name.

Reply:
Sorry I was referring to adding the `db.name` attribute.

Reply:
No, it doesn't have any concept of databases that can be used for `db.name`.